lib/service/service.go

```diff
@@ -83,9 +84,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var log = logrus.WithFields(logrus.Fields{
-	trace.Component: teleport.ComponentProcess,
-})
+var log = logrus.WithField(trace.Component, teleport.ComponentProcess)
```

Comment:
looks like there are still a few references to this package-global logger instead of `process.log` below.
remove this var and fix any compilation errors below.